zerver/lib/home.py

```diff
@@ -216,4 +216,12 @@ def build_page_params_for_home_page_load(
             request_language
         )
 
+    if settings.OSS_SHOWCASE:
+        realm = user_profile.realm
+        page_params["plan_type"] = realm.get_plan_type
+        if realm.oss_showcase == 1:
+            page_params["current_oss_showcase_value"] = "hidden"
+        else:
+            page_params["current_oss_showcase_value"] = "visible"
```

Comment:
The issue is with this block of code.
I think at most places we don't this type of conversion, although recently there are some settings that use this "string" value for APIs. But I don't think that fits here. You can remove this complete block and directly use `page_params.realm_oss_showcase_policy` on the frontend side. And follow as we do for other similar settings like `email_address_visibility_values`.